x-pack/elastic-agent/pkg/agent/program/testdata/constraints_config-filebeat.yml

```diff
@@ -8,7 +8,7 @@ filebeat:
     index: logs-generic-default
     processors:
       - add_fields:
-          target: "stream"
+          target: "dataset"
```

Comment:
I tested this and it does not work as expected. The result is:

```
          "dataset" : {
            "type" : "metrics",
            "dataset" : "system.cpu",
            "namespace" : "default"
          },
```

But it should be
```
          "dataset" : {
            "type" : "metrics",
            "name" : "system.cpu",
            "namespace" : "default"
          },
```

I think the reason is on line 14 where it still states `dataset` instead of `name`.

Reply:
oh right